daily-programming/2024-05-31/main0531.cpp

```diff
@@ -0,0 +1,15 @@
+#include <iostream>
+using namespace std;
+
+int main() {
+  long a,b,x;
+  cin >> a >> b >> x;
+  long ax,bx;
+  ax = a/x;
+  bx = b/x;
+  if(a%x == 0){
+    cout << bx - ax + 1 << endl;
+  } else{
+    cout << bx - ax << endl;
+  }

```

Comment:
基本的なアルゴリズムは、自分もこんな感じで考えると思います。
ただ、もう少し良い書き方があるので、こんな感じはどうでしょう？
```suggestion
  long ans = bx - ax;
  if(a%x == 0){
    ans += 1;
  }
  cout << ans << endl;
```